### ISSUE TITLE
Telemetry for aws hub

### ISSUE THREAD

**Issue by TomAugspurger** [pangeo-data/pangeo-cloud-federation#634]
The gcp-specific parts in https://github.com/pangeo-data/pangeo-cloud-federation/pull/629/files and https://github.com/pangeo-data/pangeo-cloud-federation/pull/626/files should probably be moved to `pangeo-deploy/values.yaml` to enable it for other hubs (AWS) rather than just the GCP one.

(cc @scottyhq)

**Comment by scottyhq** [comment]
thanks @TomAugspurger!   

> should probably be moved to pangeo-deploy/values.yaml to enable it for other hubs (AWS) rather than just the GCP one.

@salvis2 - can you take a pass at this?

**Comment by salvis2** [comment]
Yeah I can do that.

**Comment by salvis2** [comment]
@TomAugspurger you are still doing some modifications to how Prometheus / Grafana are set up on the Google hub. Should I wait until you've settled on a nice configuration for that, or just move a lot of the settings to `values.yaml`?

**Comment by TomAugspurger** [comment]
Yeah, sorry hold on this for now. I'll hopefully have something later today.

The tldr is that we'll have a separate helm deployment / namespace for
metrics I think.
